### PR TITLE
Revdep check

### PR DESCRIPTION
**Pull request by krlmlr**
preparing for CRAN release

### CODE REVIEW

File: revdep/README.md [hunk 1]
```diff
@@ -0,0 +1,1216 @@
...
+|BH        |   |1.60.0-1    |2015-12-28 |CRAN (R 3.2.3)                   |
+|DBI       |   |0.3.1       |2014-09-24 |CRAN (R 3.2.3)                   |
+|knitr     |   |1.12.3      |2016-01-22 |CRAN (R 3.2.3)                   |
+|Rcpp      |   |0.12.3      |2016-01-10 |CRAN (R 3.2.3)                   |
+|rmarkdown |   |0.9.5       |2016-02-22 |CRAN (R 3.2.3)                   |
+|RSQLite   |   |1.0.9002    |2016-03-16 |local (rstats-db/RSQLite)        |
+|testthat  |   |0.11.0.9000 |2016-03-16 |Github (hadley/testthat@a102b89) |
+
+# Check results
+55 checked out of 0 dependencies
```
Comment (krlmlr) [inline revdep/README.md]: @hadley: This looks weird.

Comment (hadley) [inline revdep/README.md]: Yeah, it's a bug :/

File: revdep/README.md [hunk 2]
```diff
@@ -0,0 +1,1216 @@
...
+
+1 error  | 0 warnings | 0 notes
+
+```
+checking whether package ‘CITAN’ can be installed ... ERROR
+Installation failed.
+See ‘/nas/muelleki/git/RSQLite/revdep/checks/CITAN.Rcheck/00install.out’ for details.
+```
+
+## DBI (0.3.1)
```
Comment (krlmlr) [inline revdep/README.md]: No idea why CRAN DBI is installed here.

**Comment by hadley** [comment]
Given the large number of install failures, I think you better check out the individual install logs.  Devtools doesn't help much yet.

**Comment by hadley** [comment]
We also probably need to check bioconductor packages, since they're heavy users of this package.

File: revdep/README.md (hunk 2 above)
Comment (hadley) [inline revdep/README.md]: Because DBI uses RSQLite for the examples

**Comment by daynefiler** [comment]
Please take a look at #145 before releasing to CRAN -- I think there is a bug with data classes.

**Pull request merged**